test/integration/container/multimaster_test.go

```diff
@@ -293,15 +303,15 @@ func TestMultimasterSetup(t *testing.T) {
 			for _, apiServerArg := range expectedApiServerArgs {
 				log.Infof("Checking api server arg (%s) on node%d", apiServerArg, i)
 				run(t, "footloose",
-					"-c", "../../../examples/footloose/centos7/docker/multimaster.yaml",
+					"-c", "../../../examples/footloose/"+node_os+"/docker/multimaster.yaml",
```

Comment:
Same

Reply:
See my next comment down as it applies to this line as well.  Very disturbing!

Reply:
Fixed.